### ISSUE TITLE
ResNet50v1.5 MxNet

### ISSUE THREAD

**Issue by Meeeee6623** [NVIDIA/DeepLearningExamples#1026]
Related to **ResNet50v1.5 with MxNet** 
*MxNet/Classification/RN50v1.5/*

**Updated model not used for testing**
When using the synthetic dataloader with MxNet, a model is updated, and training accuracy improves normally. However, testing accuracy remains at roughly .02 throughout the training process. This seems to indicate that the trained model is not being used for validation epochs, leading to the model having an accuracy close to blind guessing. 

**To Reproduce**
1. Follow the Quick Start guide for ResNet50v1.5 with MxNet
2. run the ./runner wrapper with the --data-backend flag set to synthetic.

**Expected behavior**
Training should perform as expected, with speeds roughly the same as reported in the README. However, all validation epochs will report an extremely low accuracy, roughly equivalent to blind guessing. 

**Environment**
Please provide at least:
* ResNet-50 MXNet NGC container built from MxNet/Classification/RN50v1.5/
* 2x Tesla V100-32GB):
* CUDA driver version: 470.42.01
